app/javascript/src/components/Notes/Create/index.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+
+import { Check } from "@bigbinary/neeto-icons";
+import { Pane, Typography, Button } from "@bigbinary/neetoui/v2";
+import { Input, Textarea, Select } from "@bigbinary/neetoui/v2/formik";
+import { Formik, Form } from "formik";
+import PropTypes from "prop-types";
+
+import formInitialValue from "constants/formInitialValues";
+import formValidationSchema from "constants/formValidationSchemas";
+
+import { CONTACT_OPTIONS, TAG_OPTIONS } from "../constants";
+
+function CreateNote({ showPane, setShowPane, createNote }) {
+  const onSubmit = values => {
+    const note = {
+      title: values.title.trim(),
+      description: values.description.trim(),
+      createdAt: Date.now().toString(),
+      user: { imageUrl: "https://i.pravatar.cc/300" },
+    };
+    createNote(note);
+  };
+
+  return (
+    <Pane onClose={() => setShowPane(false)} isOpen={showPane}>
+      <Formik
+        initialValues={formInitialValue.createNoteForm}
+        onSubmit={onSubmit}
+        validateOnChange={false}
+        validateOnBlur={false}
+        validationSchema={formValidationSchema.createNoteForm}
+      >
+        {({ isSubmitting }) => (
+          <Form>
+            <Pane.Header>
+              <Typography style="h2" weight="semibold">
+                Add New Note
+              </Typography>
+            </Pane.Header>
+            <Pane.Body className="w-full">
+              <div className="w-full flex flex-col ">
+                <Input
+                  name="title"
+                  type="text"
+                  label="Title"
+                  placeholder="Enter a title"
+                  className="my-4"
+                />
+                <Textarea
+                  name="description"
+                  label="Description"
+                  placeholder="Enter note description"
+                  rows={1}
+                  className="my-4"
+                />
+                <Select
+                  label="Assigned Contact"
+                  name="contact"
+                  placeholder="Select Role"
+                  options={CONTACT_OPTIONS}
+                  className="my-4"
+                />
+                <Select
+                  label="Tags"
+                  name="tags"
+                  placeholder="Select Tags"
+                  options={TAG_OPTIONS}
+                  className="my-4"
+                  isMulti
+                />
```

Comment:
Rather than using `my-4` on each individual block, please use `space-y-*` utils on the parent wrapper.